tests/provisioning-manifest-snapshot.yml

```diff
@@ -1,7 +1,30 @@
+- addMavenRepository: 'https://store.jahia.com/nexus/content/repositories/jahia-public-app-store@id=JahiaStore'
+- addMavenRepository: 'https://devtools.jahia.com/nexus/content/groups/public/@snapshots@noreleases@id=JahiaSnapshot'
+
 - addMavenRepository: 'https://devtools.jahia.com/nexus/content/groups/internal@id=jahia-internal@snapshots'
   username: ${env:NEXUS_USERNAME}
   password: ${env:NEXUS_PASSWORD}
 
+# This modules were added during the switch from using jahia-discovery to jahia-ee
+- installBundle: 
+  - 'mvn:org.jahia.modules/press/3.1.0'
```

Comment:
we're missing the digitall module here too

Reply:
The tests are passing without it, so I'd be in favor of not adding it except if required by the tests.